### PR TITLE
Added ASCAT data

### PR DESCRIPTION
**Pull request by Tomcariello**
I have added ASCAT (ocean wind data) geoJSON files. These files contain the data needed to implement a particle flow rendering. These will be made available to individuals working towards a particle flow solution in Openlayers.

### CODE REVIEW

**Review by minniewong — COMMENTED** [review]
Could you briefly describe the 14 files, are they different dates? Also looking at the filenames, I guess they are actually from METOP-B, so you will want to change the link to https://podaac.jpl.nasa.gov/dataset/ASCATB-L2-Coastal

**Comment by Tomcariello** [comment]
> Could you briefly describe the 14 files, are they different dates? Also looking at the filenames, I guess they are actually from METOP-B, so you will want to change the link to https://podaac.jpl.nasa.gov/dataset/ASCATB-L2-Coastal

Link is updated & file description has been updated based on the info provided by Joe. Thanks.

File: data/vector-flow/ascat/readme.md
Comment (minniewong) [inline data/vector-flow/ascat/readme.md:1]: Can you also change this to MetOp-B?

File: data/vector-flow/ascat/readme.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ASCAT DATA
+This dataset contains operational near-real-time Level 2 coastal ocean surface wind vector retrievals from the Advanced Scatterometer (ASCAT) on MetOp-A.
```
Comment (minniewong) [inline data/vector-flow/ascat/readme.md]: Can you change this to MetOp-B

**Review by minniewong — APPROVED** [review]
Besides the additional minor change to MetOp-B, everything looks great, thanks!

**Pull request merged**